pootle/apps/pootle_store/forms.py

```diff
@@ -308,11 +309,18 @@ def save(self, *args, **kwargs):
                     unit.reviewed_by = None
                     unit.reviewed_on = None
                     unit._log_user = self.user
+                suggestion = self.cleaned_data["suggestion"]
+                user = (
+                    suggestion.user
+                    if suggestion
+                    else self.user)
```

Comment:
So if we have suggestion, shall we set `suggestion.user` as `submitter` and `self.user` as the `reviewer` for the submission below?